internal/provisioner/plain/plain.go

```diff
@@ -0,0 +1,36 @@
+package plain
+
+import (
+	helmclient "github.com/operator-framework/helm-operator-plugins/pkg/client"
+	"github.com/operator-framework/rukpak/internal/provisioner/plain/controllers"
+	"github.com/operator-framework/rukpak/internal/source"
+	"github.com/operator-framework/rukpak/internal/storage"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	crfinalizer "sigs.k8s.io/controller-runtime/pkg/finalizer"
+)
+
+// TODO: Minimize signature
+func SetupProvisionerWithManager(ns string, mgr manager.Manager, bundleStorage storage.Storage, bundleFinalizers crfinalizer.Finalizers, unpacker source.Unpacker) error {
```

Comment:
So this gets into some `Go` intricacies. Since `manager.Manager` is an interface, a pointer to that interface isn't accepted here. We can't even call the interface's methods. IIRC, an interface is, under the hood, a pointer to its implementation so we are doing what you're asking already. 

Referenced: https://stackoverflow.com/questions/44370277/type-is-pointer-to-interface-not-interface-confusion

Reply:
Interesting, thanks for clearing that up.